resources/user-agents/apps/naver/naver-android.json

```diff
@@ -0,0 +1,57 @@
+{
+  "division": "Naver #MAJORVER#.#MINORVER# for Android",
```

Comment:
I did end up splitting the Naver android patterns into their own file so I could add some older versions without having to add the android patterns to the naver file that has more versions for iOS.